src/helm/benchmark/static/benchmarking.js

```diff
@@ -1155,38 +1266,59 @@ $(function () {
 
     // Links
     if (table.links.length > 0) {
-      $output.append(renderItems(table.links.map((link) => {
-        return $('<a>', {href: link.href}).append(link.text);
-      })));
+      $output.append(
+        renderItems(
+          table.links.map((link) => {
+            // replace everything before benchmark output, as it is incorrect
+            let myLink = link.href;
+            const delimiter = "benchmark_output";
+            const replacement = "https://storage.googleapis.com/crfm-helm-public/";
```

Comment:
`const replacement = BENCHMARK_OUTPUT_BASE_URL;`

`BENCHMARK_OUTPUT_BASE_URL` is a global constant.